README.md

```diff
@@ -22,6 +22,13 @@ The VMware GuestInfo datasource can be installed on any Linux distribution where
 curl -sSL https://raw.githubusercontent.com/vmware/cloud-init-vmware-guestinfo/master/install.sh | sh -
 ```
 
+To install a specific version, execute the following:
+
+```shell
+export GIT_REF=v1.4.1
```

Comment:
Love this addition! One thing -- could we make it a little more flexible to support not just different versions from the primary repo, but from any fork as well? Such as [this example](https://github.com/kubernetes-sigs/image-builder/blob/212059706cb1b319c5dcdc39752048fb41e2cb54/images/capi/packer/ova/packer-common.json#L8-L10) from the Kubernetes image builder. It might look like this below:

---

To install a specific version, use the following environment variables to customize the source of the install script:

| Name | Description | Default |
|---|---|---|
| `GIT_USER` | The name of a GitHub user or organization. | `vmware` |
| `GIT_REPO` | The name of the GitHub repository. | `cloud-init-vmware-guestinfo` |
| `GIT_SLUG` | A `/` delimited combination of a GitHub user/organization and repository. | `${GIT_USER}/${GIT_REPO}` |
| `GIT_REF` | A git branch name, tag name, or commit ID. | `v1.4.1` |
| `GIT_URL` |  The full path to the datasource installation script. | `${GIT_SLUG}/${GIT_REF}/install.sh` |

The following command uses the above environment variables in such a way that if an environment variable is already set, its existing value will be used, otherwise the value will default to the value from the above table:

```shell
GIT_USER="${GIT_USER:-vmware}" \
GIT_REPO="${GIT_REPO:-cloud-init-vmware-guestinfo}" \
GIT_SLUG="${GIT_SLUG:-https://raw.githubusercontent.com/${GIT_USER}/${GIT_REPO}}" \
GIT_REF="${GIT_REF:-v.1.4.1}" \
GIT_URL="${GIT_URL:-${GIT_SLUG}/${GIT_REF}/install.sh}" \
curl -sSL "${GIT_URL}" | sh -
```

Reply:
I am also fine accepting this as-is if you think the above overcomplicates things.

Reply:
One small nit though would be that you should not need to export the env var:

```
GIT_REF="v1.4.1" &&
curl -sSL "https://raw.githubusercontent.com/vmware/cloud-init-vmware-guestinfo/${GIT_REF}/install.sh"
```

The above will keep the shell free of an additional, exported env var; almost always a good thing.

Reply:
A guide on how to install from a fork might be suitable for `CONTRIBUTING.md`. As of now, I think whoever decides to fork and install their own version should be deep enough into the weeds to figure out the installation.